rpmconf/Install/Util/utilfunc.sh

```diff
@@ -2355,6 +2411,8 @@ getInstallPackages() {
 
     if [ $i = "zimbra-license-tools" ]; then
       response="yes"
+    elif [ $i = "zimbra-patch" ]; then
```

Comment:
zimbra-patch should only be installed in STORE_SELECTED = "yes"

Reply:
Done the changes.